.buildkite/x-pack/pipeline.xpack.auditbeat.yml

```diff
@@ -26,46 +26,135 @@ env:
   ASDF_MAGE_VERSION: 1.15.0
 
 steps:
+  - group: "Mandatory Tests"
+    key: "mandatory-tests"
+    steps:
+      - label: ":linux: Ubuntu Unit (MODULE) Tests"
+        key: "mandatory-linux-unit-test"
+        command: "cd $BEATS_PROJECT_NAME && mage build unitTest"
+        env:
+          MODULE: $MODULE
```

Comment:
I think @dliappis changed to having a script loading the proper module.

Reply:
> I think @dliappis changed to having a script loading the proper module.

Correct. @pazone plz take a look at https://github.com/elastic/beats/pull/38827/files#diff-87cc9e7aecd3c1a6382e951a1b17429443aa3f12387328521fa06b2c8f83a814R52-R55 on how to do this at the command level.

Reply:
That's nice! Updated the PR according to the example.

Reply:
I slightly changed it. `$PWD` didn't work because it's the workspace dir.

Reply:
Not sure if it works correctly https://buildkite.com/elastic/beats-xpack-auditbeat/builds/1307#018ec9f2-4ca7-4b80-bda0-42bdf681c724/10717-10728

Reply:
> Also copied `OSS_MODULE_PATTERN` and `XPACK_MODULE_PATTERN` to `changesets.sh` to make it independent of `common.sh`

This fix was already merged yesterday in main (and backported) see https://github.com/elastic/beats/pull/38830

Reply:
> I slightly changed it. `$PWD` didn't work because it's the workspace dir.

Good catch! Indeed, as hinted by the var name, the parameter should be the path to the project: https://github.com/elastic/beats/blob/main/.buildkite/scripts/changesets.sh#L49 (I also fixed it in my PR)

That said, currently the script (see the conditional [here](https://github.com/elastic/beats/blob/main/.buildkite/scripts/changesets.sh#L71-L79)), due [this issue](https://github.com/elastic/ingest-dev/issues/2993), to will basically set module to `AWS` only for xpack-metricbeat, and to `kubernetes` for everything else.

Reply:
Ok, then we aligned here, right?